test/getGrist.ts

```diff
@@ -183,6 +192,28 @@ export class GristUtils extends GristWebDriverUtils {
     await this.waitForServer();
   }
 
+  public async sendActions(actions: UserAction[]) {
+    const result = await driver.executeAsyncScript(`
```

Comment:
grep `grist-core` for examples of calling `executeAsyncScript` in a somewhat neater way (though no big deal if it proves a problem).